src/Sulu/Bundle/CategoryBundle/DependencyInjection/SuluCategoryExtension.php

```diff
@@ -55,6 +58,9 @@ public function prepend(ContainerBuilder $container)
                 [
                     'exception' => [
                         'codes' => [
+                            CategoryNotFoundException::class => 404,
+                            CategoryKeyNotUniqueException::class => 409,
+                            CategoryNameMissingException::class => 409,
```

Comment:
That's not a `409`, would rather say a `400`, because some data is missing, and not necessarily conflicting with anything else.